plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/AbstractProjectTest.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Common infrastructure for workbench-based tests that create a single project.
  */
-public class AbstractProjectTests {
+public class AbstractProjectTest {
```

Comment:
I'm surprised the name of the superclass matters, but c'et la vie. However this class should be abstract (or renamed).